app/models/katello/content_view.rb

```diff
@@ -95,11 +95,13 @@ class ContentView < Katello::Model
                                :repository_import,
                                :repository_export_syncable])
     }
-    scope :ignore_generated, -> {
-      where.not(:generated_for => [:repository_export,
-                                   :repository_import,
-                                   :library_export_syncable,
-                                   :repository_export_syncable])
+    scope :ignore_generated, ->(include_library_generated = false) {
```

Comment:
```suggestion
    scope :ignore_generated, ->(ignore_library_generated = false) {
```

include_library_generated made me think that it wasn't ignoring library generated repos.

Reply:
So when ignore_library_generated is true we return library_export and library_import as part of the ignore_generated list and vice-versa. 
With include_library_generated , I was thinking of it as including library_generated in the ignore_generated list.

Reply:
Alrighty, I could see it either way